backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -501,6 +501,8 @@ export class ContributionResolver {
         transaction.typeId = TransactionTypeId.CREATION
         transaction.memo = contribution.memo
         transaction.userId = contribution.userId
+        transaction.userGradidoID = user.gradidoID
+        transaction.userName = user.firstName + ' ' + user.lastName
```

Comment:
Please test that this fields are present

Reply:
done